scipy/fftpack/tests/test_basic.py

```diff
@@ -324,6 +324,29 @@ def test_invalid_sizes(self):
         assert_raises(ValueError, rfft, [])
         assert_raises(ValueError, rfft, [[1,1],[2,2]], -5)
 
+    # See Issue #5790
```

Comment:
@pv , @larsmans : as this is a non-code related suggestion, I'll withhold from making the change unless there are no code-related changes to be made.